src/compiler/checker.ts

```diff
@@ -10702,8 +10779,24 @@ namespace ts {
                 }
 
                 let derived = getTargetSymbol(getPropertyOfObjectType(type, base.name));
-                if (derived) {
-                    let baseDeclarationFlags = getDeclarationFlagsFromSymbol(base);
+                let baseDeclarationFlags = getDeclarationFlagsFromSymbol(base);

```

Comment:
`checkKindsOfPropertyMemberOverrides()` though I think we've resolved the issue offline.